### PR TITLE
WRN-6719: Revert ContextualPopupDecorator arrow patch

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Remove workaround code related to chrome rendering issue as patch has been applied.
chromium issue link: https://bugs.chromium.org/p/chromium/issues/detail?id=996931

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Revert https://github.com/enactjs/enact/pull/2501

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-6719

### Comments

### CODE REVIEW

**Review by SkylerBaek — APPROVED** [review]
:lgtm:

File: samples/sampler/stories/qa/ContextualPopupDecorator.js [hunk 1]
```diff
@@ -44,6 +45,15 @@ const renderSuperTallPopup = () => (
 	</div>
 );
 
+const renderSliderPopup = () => (
+	<IncrementSlider
+		style={{width: '500px'}}
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/ContextualPopupDecorator.js]:
```suggestion
		style={{width: ri.scaleToRem(500)}}
```

File: samples/sampler/stories/qa/ContextualPopupDecorator.js [hunk 2]
```diff
@@ -44,6 +45,15 @@ const renderSuperTallPopup = () => (
...
+		max={100}
+		min={0}
+		orientation="horizontal"
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/ContextualPopupDecorator.js]:
```suggestion
```

It looks same with default value, not needed.

File: samples/sampler/stories/qa/ContextualPopupDecorator.js [hunk 3]
```diff
@@ -258,4 +268,18 @@ storiesOf('ContextualPopupDecorator', module)
...
 		)
+	)
+	.add(
+		'with increment slider',
+		() => (
+			<div style={{textAlign: 'center', marginTop: ri.unit(180, 'rem')}}>
+				<ContextualPopupWithActivator
+					direction="down"
+					popupComponent={renderSliderPopup}
+					spotlightRestrict="self-only"
```
Comment (jeonghee27) [inline samples/sampler/stories/qa/ContextualPopupDecorator.js]:
```suggestion
					direction={select('direction', ['up', 'down', 'left', 'right'], Config, 'down')}
					popupComponent={renderSliderPopup}
```
If this value is not required, how about making it optional?

File: samples/sampler/stories/qa/ContextualPopupDecorator.js (hunk 1 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/ContextualPopupDecorator.js]: It's fixed

File: samples/sampler/stories/qa/ContextualPopupDecorator.js (hunk 2 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/ContextualPopupDecorator.js]: It's removed

File: samples/sampler/stories/qa/ContextualPopupDecorator.js (hunk 3 above)
Comment (juwonjeong) [reply samples/sampler/stories/qa/ContextualPopupDecorator.js]: It's fixed.

**Pull request merged**